### PR TITLE
Upgrading to latest integration sdk

### PR DESCRIPTION
**Pull request by softwarewright**

### CODE REVIEW

**Comment by softwarewright** [comment]
@erkangz I had to make some changes to some of the tests, can you validate that the changes are fine.

File: src/__tests__/validateInvocation.test.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
-import { createMockExecutionContext } from '@jupiterone/integration-sdk/testing';
+import { createMockExecutionContext } from '@jupiterone/integration-sdk-testing';
 
 import validateInvocation from '../validateInvocation';
 
 import fetchMock from 'jest-fetch-mock';
+import { ServicesClientInput } from 'src/collector/ServicesClient';
```
Comment (erkangz) [inline src/__tests__/validateInvocation.test.ts:6]: `src/` will cause problems

Comment (softwarewright) [reply src/__tests__/validateInvocation.test.ts:6]: Thanks for the heads up.

**Comment by charlieduong94** [comment]
You probs saw it in the build, but you will need to remove the `--noEmit` from the `type-check` command.

File: tsconfig.json [hunk 2]
```diff
@@ -1,17 +1,5 @@
...
-    "noUnusedLocals": true,
-    "pretty": true,
-    "esModuleInterop": true,
-    "baseUrl": ".",
-    "paths": {
-      "test": ["test"]
-    }
-  },
+  "extends": "./node_modules/@jupiterone/integration-sdk-dev-tools/config/typescript",
+  "outDir": "dist",
```
Comment (charlieduong94) [inline tsconfig.json:3]: This `outDir` needs to be placed under `compilerOptions`. `exclude` can stay where it's at though

Comment (softwarewright) [reply tsconfig.json:3]: Oh I missed that you're right, I'll update that. Good catch!

Comment (softwarewright) [reply tsconfig.json]: Fixed

**Pull request merged**